src/Logger.js

```diff
@@ -62,81 +63,109 @@ export default class Logger {
       return acc;
     }, {});
 
-    this.winston = new winston.Logger({
+    this.winston = winston.createLogger({
```

Comment:
In L52, can we adjust the default level for development? See https://github.com/molindo/molindo-node-logger/issues/17.